lib/homepage.dart

```diff
@@ -0,0 +1,111 @@
+import 'package:flutter/material.dart';
+import 'package:tutorial/constants.dart';
+
+class Homepage extends StatefulWidget {
```

Comment:
We discussed in class and decided home page is two words.  That means this widget should be HomePage and the file should be home_page.dart